comment/tests/test_api/test_serializers.py

```diff
@@ -27,23 +27,23 @@ def comment_count_test(self):
 
     def test_get_profile_model(self):
         # missing settings attrs
-        patch.object(settings, 'PROFILE_APP_NAME', None).start()
-        profile = get_profile_model()
-        self.assertIsNone(profile)
+        with patch.object(settings, 'PROFILE_APP_NAME', None):
```

Comment:
Why don't we use decorator? We can split the test if it uses patched value more than once.

Reply:
I didn't want to go that route otherwise it would refactoring a lot of tests. Some of them are written as functions as of now, which can be changed to functions and then, use decorators. I believe this can be done in a separate commit.

Reply:
as of now, I have only refactored the ones that I had to debug for this issue.

Reply:
I don't mind to keep it this way, but since we are in the subject, why don't we finish it in one time?
If you feel that you are busy, I can continue your wrok.

Reply:
If you feel that you are busy, I can continue your wrok.

No, it is not due to time constraint. I just don't like too big changes in one commit unless it is really necessary. Too many changes in one go, make it harder to review and also litter the `git` history in my opinion. 

Some tests really require refactoring as they test more than one unit at present. I would probably want to do this in a separate commit.

Reply:
No problem. I will merge this and will open a new issue for splitting unit tests in case I will have time later.